src/clients/order/index.ts

```diff
@@ -25,4 +27,14 @@ export class Order {
 		this.config.options = { ...this.config.options, ...requestOptions };
 		return create({ body, config: this.config });
 	}
+
+	/**
+   * Process Order.
+   *
+   * @see {@link https://github.com/mercadopago/sdk-nodejs/blob/master/src/examples/order/process.ts Usage Example }.
+   */
+	processOrder({ id, requestOptions }: ProcessOrderData): Promise<OrderResponse> {
```

Comment:
Seguir o mesmo padrão do get e create, não precisamos mencionar Order no nome dessa função. 
```suggestion
	process({ id, requestOptions }: ProcessOrderData): Promise<OrderResponse> {
```